packages/vue/src/components/molecules/SfCollectedProduct/SfCollectedProduct.html

```diff
@@ -0,0 +1,38 @@
+<div class="sf-collected-product">
+  <!-- @slot -->
+  <slot name="remove">
+    <SfCircleIcon color="gray-secondary" class="sf-collected-product__remove" @click="removeProduct">
+      <sf-icon icon="cross" color="white" size="8px" />
+    </SfCircleIcon>
+  </slot>
+  <div class="sf-collected-product__container">
+    <slot name="image" v-bind="{image, title}">
```

Comment:
```suggestion
    <slot name="image" v-bind="{image, title, quantity}">
```
can you create two slots? image and counter? @filrak @leomp12 wdyt?